main/client/header/navbar.js

```diff
@@ -2,15 +2,14 @@ if (Meteor.isClient) {
 
   Meteor.startup(function() {
   // Runs when the DOM is ready
-  
-  Session.set('willShowAutoSuggestion',false);
 
+    Session.set('willShowAutoSuggestion',false);
```

Comment:
willShowAutoSuggestion is returned by the helper shouldShowAutoSuggestion in searchBox template. Because the navbar is the same instance regardless of the route , i needed some way of controling that we didnt want autosuggestion in the input and rather inline search in the /projects route.